.pre-commit-config.yaml

```
@@ -33,7 +33,11 @@ repos:
   rev: v0.9.27
   hooks:
   - id: pymarkdown
-    files: docs/.*
+    # Conflicts with pyml disable, so we flag this to be fixed manually
+    args: [fix, -d, md007]
+  hooks:
+  - id: pymarkdown
+    args: [scan]
```

Is it necessary to have `hooks` twice?

I'm also curious to know an example where `fix` clashed with the tool`s own ignore syntax?

```suggestion
    args: [scan]
  - id: pymarkdown
    # Conflicts with pyml disable, so we flag this to be fixed manually
    args: [fix, -d, md007]
```

---

If I only use `fix` without extra options, the linter attempts to update the lines inside `rocm.inc.md`. I intentionally put `fix` after `scan` so that some of the updates can be applied automatically via `fix` without having to manually resolve the errors from `scan`.

---

Good point about the order. I've edited my suggestion to just be about duplicating the hooks key

---

Fixed in 069c055

---

It's strange that it tries to edit `rocm.inc.md`, that should already have been handled by the existing hooks 🤔